framework/di/Instance.php

```diff
@@ -128,7 +128,11 @@ public static function ensure($reference, $type = null, $container = null)
         }
 
         if ($reference instanceof self) {
-            $component = $reference->get($container);
+            try {
```

Comment:
Created this: https://3v4l.org/N4rbA not sure if the results are significant in any way.